action_server/src/sema4ai/action_server/_protocols.py

```diff
@@ -183,3 +191,59 @@ class ArgumentsNamespaceStart(ArgumentsNamespaceBaseImportOrStart):
     expose: bool
     expose_allow_reuse: bool
     api_key: str
+
+
+class ArgumentsNamespacePackageOrganizations(ArgumentsNamespace):
+    command: Literal["package"]
+    package_command: Literal["organizations"]
+    access_credentials: Optional[str]
+    json: bool
+
+
+class ArgumentsNamespacePackagePush(ArgumentsNamespace):
+    command: Literal["package"]
+    package_command: Literal["publish"]
+    access_credentials: Optional[str]
+    organization_id: str
+    package_path: str
+    json: bool
+
+
+class ArgumentsNamespacePackageStatus(ArgumentsNamespace):
+    command: Literal["package"]
+    package_command: Literal["state"]
+    access_credentials: Optional[str]
+    organization_id: str
+    package_id: str
+    json: bool
+
+
+class ArgumentsNamespacePackageChangelog(ArgumentsNamespace):
+    command: Literal["package"]
+    package_command: Literal["changelog"]
+    access_credentials: Optional[str]
+    organization_id: str
+    package_id: str
+    json: bool
+    change_log: str
+
+
+class ArgumentsNamespacePackagePublish(ArgumentsNamespace):
+    command: Literal["package"]
+    package_command: Literal["publish"]
+    organization_name: Optional[str]
+    access_credentials: Optional[str]
+    package_path: str
+    change_log: str
+
+
+class ArgumentsNamespaceCloud(ArgumentsNamespace):
+    command: Literal["cloud"]
+    cloud_command: Literal["login"]
+
+
+class ArgumentsNamespaceCloudlogin(ArgumentsNamespace):
+    command: Literal["cloud"]
+    cloud_command: Literal["login"]
+    hostname: str
```

Comment:
Question: I see that `access_credentials` could be used here or could be passed on to each of the other commands explicitly. Should the same be true for the `hostname`?